### PR TITLE
Updates for claim datatype and warranty/insurance claim details

### PR DESCRIPTION
**Pull request by jbranson000**
Updates for claim datatype and warranty/insurance claim details

### CODE REVIEW

File: components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-details.schema.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Claim Process",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["financial_services"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Claim details such as claim policy ID and claim creation steps.",
```
Comment (hanessian) [inline components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-details.schema.json:18]: Let's adjust this to reflect the deprecation of the toolusage...

Claim policy ID, police report, and incident date.

File: components/datatypes/industry-verticals/claim.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          "title": "Policy ID",
+          "type": "string",
+          "description": "Policy Identifier."
+        },
+        "xdm:benefitAmount": {
+          "title": "Benefit Amount",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "description": "Captures the benefit amount."
+        },
+        "xdm:steps": {
```
Comment (mjasrotia) [inline components/datatypes/industry-verticals/claim.json:44]: Please remove steps as we previously discussed. We can add this later if required

File: components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-details.schema.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-insurance-claim-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Claim Process",
```
Comment (mjasrotia) [inline components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-details.schema.json:10]: Should this be "Insurance Claim Process" to be more specific  ?

File: components/datatypes/industry-verticals/claim.json (hunk 2 above)
Comment (jbranson000) [reply components/datatypes/industry-verticals/claim.json:44]: There are still instances - particularly with Analytics - where we will use this.  Can we leave it in as deprecated for now?

File: components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-details.schema.json (hunk 3 above)
Comment (jbranson000) [reply components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-details.schema.json:10]: Renamed

File: components/mixins/experience-event/industry-verticals/experienceevent-warranty-claim-details.schema.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-warranty-claim-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Claim Process",
```
[the remark this answers is not in the document]
Comment (jbranson000) [reply components/mixins/experience-event/industry-verticals/experienceevent-warranty-claim-details.schema.json:10]: renamed

File: components/datatypes/industry-verticals/claim.json (hunk 2 above)
Comment (hanessian) [reply components/datatypes/industry-verticals/claim.json:44]: We should only deprecate things that once existed as stable, not introduce them before they ever existed as deprecated. Is it ok if we remove this for now, @jbranson000 ?

Comment (hanessian) [reply components/datatypes/industry-verticals/claim.json:44]: once we remove that we can push this PR through :)

Comment (jbranson000) [reply components/datatypes/industry-verticals/claim.json]: Of course - that'll be fine.  I went ahead and removed it.

File: components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-process.schema.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["financial_services"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Claim details including policy ID, police report, and incident date.",
+  "definitions": {
+    "claim-details": {
+      "properties": {
+        "xdm:insurance-claim": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-process.schema.json:22]: We do not support "-" in property names. I am changing this.

File: components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-process.schema.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+            },
+            "xdm:incidentLocation": {
+              "title": "Incident Location",
+              "$ref": "https://ns.adobe.com/xdm/common/address",
+              "description": "Location where the claim incident occurred."
+            },
+            "xdm:type": {
+              "title": "Type",
+              "type": "string",
+              "meta:enum": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/industry-verticals/experienceevent-insurance-claim-process.schema.json:55]: Please add the enums

File: components/datatypes/industry-verticals/claim.json [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+          "title": "Steps",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/tool-usage",
+          "description": "Extension of toolUsage data type.",
+          "meta:status": "deprecated"
```
Comment (mjasrotia) [inline components/datatypes/industry-verticals/claim.json:48]: I guess this was added back by mistake. can we remove this ?

File: components/datatypes/industry-verticals/claim.xample.1.json [hunk 8]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mjasrotia) [inline components/datatypes/industry-verticals/claim.xample.1.json:1]: Please correct the file name xample ==> example

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**